CHANGELOG.md

```diff
@@ -1,3 +1,145 @@
+# 11.0.0-rc.2 "vicuna-valley" (2020-11-10)
```

Comment:
I won't block on it, but `vicuna` isn't a material 😃

Reply:
https://www.scabal.com/en/story/vicuna-luxurious-fabric-world/ 🥺